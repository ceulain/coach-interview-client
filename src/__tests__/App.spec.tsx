import React from "react";
import renderer from "react-test-renderer";
import App from "../App";

describe("App component", () => {
  test("should return component", () => {
    expect(renderer.create(<App />)).toMatchSnapshot();
  });
});
