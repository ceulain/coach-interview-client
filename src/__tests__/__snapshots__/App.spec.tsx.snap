// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should return component 1`] = `
<div
  className="App"
>
  <h1>
     Hello, Azerty! 
  </h1>
</div>
`;
